grc/core/blocks/block.py

```diff
@@ -539,6 +541,26 @@ def is_virtual_sink(self):
     def is_virtual_source(self):
         return self.key == 'virtual_source'
 
+    def is_deprecated(self):
+        """
+        Check whether the block is deprecated.
+
+        For now, we just check the category name for presence of "deprecated".
+
+        As it might be desirable in the future to have such "tags" be stored
+        explicitly, we're taking the detour of introducing a property.
+        """
+        if not self.category:
+            return False
+
+        try:
+            return any("deprecated".casefold() in cat.casefold()
+                       for cat in self.category)
+        except Exception as exception:
+            print(exception.message)
```

Comment:
This feels wrong, but, yes, there is no logger defined for this module. If you wanted to go the extra mile you could add one and report this through it. There is logging configuration...

Reply:
I'll opt to not do this at this time, though, and do that separately, and then consistently for the whole module. Let me add a `#fixme`, however.